draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -179,6 +179,8 @@ The list of member zones is specified as a collection of member nodes, represent
 The names of member zones are represented on the RDATA side (instead of as a part of owner names) of a PTR record, so that all valid domain names may be represented regardless of their length [@!RFC1035].
 This PTR record MUST be the only record in the PTR RRset with the same name.
 More than one record in the RRset denotes a broken catalog zone which MUST NOT be processed (see (#generalrequirements)).
+A clear error message stating that more than one PTR in the RRset caused the
+broken catalog, with the owner name of that RRset, MUST be logged.
```

Comment:
```suggestion
The reason a catalog zone is considered broken MUST always be communicated clearly to the operator.
```

I'm open to other wording, but "logged" prescribes too much about how one operates their services.

Reply:
I disagree with @Habbie, as "logged" in my mind does not necessarily mean a written log message; any "log event" (e.g. web hook triggering an alert, ...) would also qualify. But I don't mind more general phrasing.

When keeping this, perhaps add `(e.g. through a log message)`, to not falsely imply that "communicated" means sending some kind of message.

Reply:
Okay, combining all the feedback, I got:
```suggestion
The reason a catalog zone is considered broken SHOULD always be communicated clearly to the operator (e.g. through a log message).
```